src/js/components/pipelines/HoverField.jsx

```diff
@@ -110,15 +117,41 @@ var HoverField = React.createClass({
     this.setState({fieldDef: fieldDef});
   },
 
+  sortValues: function(evt) {
+  // will need to clean up this code
+  var valuesInc = this.state.valuesInc,
+      props = this.props,
+      field = props.def,
+      id = props.dsId,
+      incOrDec = null;
+
+  if (valuesInc == null) {
+    // first click default: increasing
+    valuesInc = true;
+  } else {
+    valuesInc = !valuesInc;
+  }
+  this.setState({ valuesInc : valuesInc });
```

Comment:
Be sure to run `npm run lint` to catch errors in code style like the additional spaces here.